core/txpool/legacypool/legacypool_test.go

```diff
@@ -3299,7 +3168,7 @@ func TestPoolMultiAccountBatchInsertRace(t *testing.T) {
 			pending = pool.Pending(txpool.PendingFilter{})
 			total = len(pending)
 
-			_ = pool.Locals()
+			// _ = pool.Locals()
```

Comment:
Why was this commented out?

Reply:
I should have removed it instead of commenting out.
Context: https://github.com/ethereum/go-ethereum/pull/30559 locals-tracking was removed from transaction pool.